### PR TITLE
Add image sensor 

### PR DESCRIPTION
**Pull request by AritroSaha10**
Wanted to be able to access images from the panel camera / image sensor. Tried to implement it in a way that it matches the current implementation of other types of devices (sensors, garage door, etc.), so it should work like any other device (except for a custom `get_images()` command to fetch all the taken images).

### CODE REVIEW

**Comment by AritroSaha10** [comment]
Hi, thank you for the detailed feedback! I've fixed up the mypy errors, and running the `pre-commit` task seems to return all OK on my side. Please let me know whether there is anything else to fix

**Comment by elahd** [comment]
Awesome. I made some additional changes to ensure that things are more in line with the other sensors. Are you able to check out these changes and confirm that this still works as expected?

Also, what's present in the image_urls object? Is it just a plain list of URLs, like ["http://domain.com/photo1.jpg","http://domain.com/photo2.jpg"]?

Lastly, do you need to be logged in to Alarm.com to view these images, or can they be viewed without an active session?

**Comment by AritroSaha10** [comment]
Hi, just tested it and it stills works as expected after a quick fix. 

For the image_urls object, it returns an output similar to this:

```
[
    {
      "id": "<id>",
      "type": "image-sensor/image-sensor-image",
      "attributes": {
        "image": "<base64 string of image>",
        "imageSrc": "/web/History/ImageViewer.ashx?correlated_id=<id>&frame=1&customer_id=<id>",
        "description": "Peek-In Now image",
        "timestamp": "2022-02-03T00:02:44+00:00"
      },
      "relationships": {
        "imageSensor": {
          "data": {
            "id": "<id>",
            "type": "image-sensor/image-sensor"
          }
        }
      }
    },
    ...
]
```

Basically, it returns some metadata as well as the full-resolution image as a base64 string.

File: README.md [hunk 1]
```diff
@@ -35,20 +35,21 @@ See `examples/basic_sensor_data.py` for a basic usage example.
...
-|Locks|`desired_state`|lock, unlock|
-|Garage Door|(none)|open, close|
+| Device Type  | Notable Attributes                  | Actions                               |
+| ------------ | ----------------------------------- | ------------------------------------- |
+| System       | `unit_id`                           | (none)                                |
+| Partition    | `uncleared_issues`, `desired_state` | arm away, arm stay, arm night, disarm |
+| Sensors      | `device_subtype`                    | (none)                                |
+| Locks        | `desired_state`                     | lock, unlock                          |
+| Garage Door  | (none)                              | open, close                           |
+| Image Sensor | `image_urls`                        |                                       |
```
Comment (AritroSaha10) [inline README.md:45]:
```suggestion
| Image Sensor | `image_urls`                        | peek_in                               |
```

**Comment by elahd** [comment]
@AritroSaha10 Got it, thanks for that. In that case, I think it's better for ADCImageSensor to return a sanitized object instead of just a raw dump from the API. That way we can keep the pyalarmdotcomajax response predictable and stable even if the Alarm.com API changes in the future. I submitted changes along that time. Would you be able to test once more?

**Comment by AritroSaha10** [comment]
Still works properly for me.

**Pull request merged**